WcaOnRails/config/locales/en.yml

```diff
@@ -630,6 +632,10 @@ en:
         on_the_spot_registration:
           "true": "On the spot registrations will be accepted"
           "false": "On the spot registrations WILL NOT be accepted"
+        free_entry_text:
```

Comment:
Quotes are not necessary here, the above `"true"` and `"false"` are quoted because normally they would be boolean keywords.